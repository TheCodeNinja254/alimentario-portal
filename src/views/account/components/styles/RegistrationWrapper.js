import { styled } from "@material-ui/styles";

const RegistrationWrapper = styled("div")(({ theme }) => ({
  backgroundColor: theme.palette.primary.light,
}));

export default RegistrationWrapper;
