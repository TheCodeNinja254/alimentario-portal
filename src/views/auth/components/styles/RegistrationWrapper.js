import { styled } from "@material-ui/styles";

const RegistrationWrapper = styled("div")(({ theme }) => ({
  backgroundImage: "url(/images/bg.jpg)",
  backgroundRepeat: "no-repeat",
  backgroundColor:
    theme.palette.type === "light"
      ? theme.palette.grey[50]
      : theme.palette.grey[900],
  backgroundSize: "cover",
  height: "100%",
  backgroundPosition: "center",
  opacity: 0.9,
}));

export default RegistrationWrapper;
