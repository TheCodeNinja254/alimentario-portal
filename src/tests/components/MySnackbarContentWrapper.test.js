import React from "react";
import renderer from "react-test-renderer";
import { ThemeProvider } from "@material-ui/core/styles";
import DefaultTheme from "../../theme";
import MySnackbarContentWrapper from "../../components/MySnackbarContentWrapper";

describe("<MySnackbarContentWrapper />", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <ThemeProvider theme={DefaultTheme}>
          <MySnackbarContentWrapper
            variant="danger"
            variantText="dangerText"
            message="Sorry an error occurred"
          />
        </ThemeProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
