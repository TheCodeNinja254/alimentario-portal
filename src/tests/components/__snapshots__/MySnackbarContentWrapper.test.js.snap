// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MySnackbarContentWrapper /> renders correctly 1`] = `
<div
  aria-describedby="client-snackbar"
  className="MuiPaper-root MuiSnackbarContent-root makeStyles-snackbarContent-6 MuiPaper-elevation6"
  role="alert"
>
  <div
    className="MuiSnackbarContent-message"
  >
    <div
      className="MuiBox-root MuiBox-root-18 makeStyles-message-10"
      id="client-snackbar"
    >
      <p
        className="MuiTypography-root makeStyles-dangerText-15 makeStyles-textSize-11 MuiTypography-body1"
      >
        Sorry an error occurred
      </p>
    </div>
  </div>
  <div
    className="MuiSnackbarContent-action"
  >
    <button
      aria-label="close"
      className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root makeStyles-icon-7"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
          />
        </svg>
      </span>
    </button>
  </div>
</div>
`;
